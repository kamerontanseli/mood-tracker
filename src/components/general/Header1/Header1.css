.Header1 {
  font-weight: 300;
  color: #333;
}