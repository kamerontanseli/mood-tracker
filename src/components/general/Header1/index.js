import "./Header1.css";
import React from "react";

const Header1 = props => <h1 className="Header1" {...props} />

export default Header1;
