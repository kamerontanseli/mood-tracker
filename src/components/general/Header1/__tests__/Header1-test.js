import React from 'react'
import renderer from 'react-test-renderer'
import Header1 from '..'

describe('render', () => {
  it('should render', () => {
    const snapshot = renderer.create(
      <Header1>Hello world</Header1>
    );
    expect(snapshot).toMatchSnapshot()
  })
})