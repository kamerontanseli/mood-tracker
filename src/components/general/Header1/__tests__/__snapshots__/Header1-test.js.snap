// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render 1`] = `
<h1
  className="Header1"
>
  Hello world
</h1>
`;
